api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxPvlsCohortQueries.java

```diff
@@ -116,10 +111,9 @@ public CohortDefinition getPatientsWithViralLoadSuppression() {
 		String mappings = "startDate=${startDate},endDate=${endDate},location=${location}";
 		cd.addSearch("supp",
 		    EptsReportUtils.map(hivCohortQueries.getPatientsWithSuppressedViralLoadWithin12Months(), mappings));
-		cd.addSearch("baseCohort", EptsReportUtils.map(genericCohortQueries.getBaseCohort(), mappings));
 		cd.addSearch("onArtLongEnough",
 		    EptsReportUtils.map(getPatientsWhoAreMoreThan3MonthsOnArt(), "onDate=${endDate},location=${location}"));
-		cd.setCompositionString("supp AND baseCohort AND onArtLongEnough");
```

Comment:
@ningosi @edrisse shouldn't the baseCohort still be included here to be applied to the SQL queries?

Reply:
I would highly recommend, we **include** the basecohort there as it was before. We can also have it at the report definition as well for calculations to run based on it(as it is done now), we were getting different values for TX_Curr by excluding it from the composition cohort definition @nthfloor

Reply:
Ok @edrisse please confirm with Yassin to test this by modifying a patient record so that it should be excluded by the baseCohort. We need to be sure about this. 

Looks like the SqlCohortDefinition includes the baseCohort when it [evaluates](https://github.com/openmrs/openmrs-module-reporting/blob/331fe70b9a96e5855ccc3bffe18a6fcbfbceb710/api/src/main/java/org/openmrs/module/reporting/cohort/definition/evaluator/SqlCohortDefinitionEvaluator.java#L46-L68), but there might still be a bug which we will then want to inform the community about.